### DISCUSSION TITLE
Is this compatible with the NDI 5 runtime? Or just 4?

### DISCUSSION THREAD

**Discussion by JuiceBox7535** [DistroAV/DistroAV#679] [Q&A - Support]
Is this compatible with the NDI 5 runtime? Or just 4? Thanks!

**Comment by tt2468** [comment]
It seems that some users have had success using NDI 5 with 4.9.1, while others haven't. The true compatibility for 4.9.1 is unknown.

**Reply by c0rychu** [reply]
I tried to use obs-ndi-4.9.0-macOS.pkg with ndi-runtime-5.0.10.1-macOS.pkg but obs-ndi cannot find the ndi-runtime. It keep asking me to download ndi-runtime-4.5.1-macOS.pkg ......

**Comment by sanyer** [comment]
Works well for me on two Windows PC and NDI 5.5

**Comment by krauspe** [comment]
Unfortunately doesn't works on windows 10 for me
Is there a fix planned ?

**Reply by sanyer** [reply]
There is an alpha build of this plugin for OBSv28 available here: https://github.com/Palakis/obs-ndi/actions/runs/2977380826
You might have more luck with this alpha, OBSv28 and NDIv5.

**Reply by krauspe** [reply]
Thank you but it doesn't work either. But meanwhile I have a problem with NDI 5 in general, even with their own tools, it's weird...

**Comment by Trouffman** [comment]
As of V4.10 this works with Runtime version 5  (tested on 5.5.2.0)

@tt2468 this can be closed

**Comment by Jaybonaut** [comment]
Can someone link to the current version of 5?

**Comment by Trouffman** [comment]
Latest NDI5 Runtime

See : https://github.com/obs-ndi/obs-ndi/discussions/831

**Reply by SomeonesVictor** [reply]
The windows link doesn't work

**Reply by Trouffman** [reply]
indeed, theses are the old links. 
Find a more up-to-date version here: https://github.com/obs-ndi/obs-ndi/discussions/831
